### ISSUE TITLE
Docs dark theme: right nav bar color difficult to read

### ISSUE THREAD

**Issue by kornerc** [samuelcolvin/pydantic#3895]
### Checks

* [x] I added a descriptive title to this issue
* [x] I have searched (google, github) for similar issues and couldn't find anything
* [x] I have read and followed [the docs](https://pydantic-docs.helpmanual.io/) and still think this is a bug

# Bug

With the dark theme of the docs, parts of the right nav bar (links to the sections before the current section) are difficult to read.
The contrast between their black and the background color is in my opinion a bit too low.

![grafik](https://user-images.githubusercontent.com/1033108/157868858-d21911b9-7b6b-456a-8718-e2a786ae09b3.png)

This is caused by the [customization](https://github.com/samuelcolvin/pydantic/blob/d7a8272d7e0c151b0bd43df596be02e0d436ebdf/docs/extra/tweaks.css#L21) of the default theme.

To fix this issue, the customization could be removed or an additional CSS-rule has to be added for the dark theme.

I tried to modify the CSS-selector myself but was not able to only match the light theme.
```css
.md-nav__link[data-md-state=blur]:not(.md-nav__link--active)
```

**Comment by samuelcolvin** [comment]
Cause is

https://github.com/samuelcolvin/pydantic/blob/d7a8272d7e0c151b0bd43df596be02e0d436ebdf/docs/extra/tweaks.css#L21-L27

Could either remove this or fix it to use a css var.

**Comment by kornerc** [comment]
I close the issue since it was fixed in #4074

![grafik](https://user-images.githubusercontent.com/1033108/179766258-18483702-6fdc-42f8-bf2c-6850c30ed2ad.png)

**Comment by samuelcolvin** [comment]
thanks for cleaning this up.
